### ISSUE TITLE
Support Request: Cannot connect using BLID

### ISSUE THREAD

**Issue by WotAGitUR** [taurgis/homebridge-iRobot-v2#22]
### Describe Your Problem:

I have a config setup with IP address, BLID and password, but the nothing connects.
The logs show its expecting the email and password even though manual config was selected.
Both Homebridge and the Roomba S9 are on the same network.

### Expected behavior:

I was expecting it to connect without needing the email/password

### Logs:

```shell
[22/11/2024, 16:11:06] [iRobot] Initializing iRobotPlatform platform...
[22/11/2024, 16:11:07] [iRobot] No email or password provided. Exiting setup
Setup Payload:
X-HM://0024B34HA5WLX
Enter this code with your HomeKit app on your iOS device to pair with Homebridge:
```

### Config:

```shell
{
    "name": "iRobot",
    "roombas": [
        {
            "name": "RobotS9",
            "blid": "xxxxxx",
            "password": "xxxxxxxx",
            "autoConfig": false,
            "ip": "192.168.3.22",
            "model": "S9",
            "multiRoom": true,
            "info": {
                "ver": 2
            }
        }
    ],
    "manualDiscovery": true,
    "lowBattery": 20,
    "offAction": "pause:dock",
    "status": "cycle:clean",
    "eveStatus": "inverted:cycle:clean",
    "bin": "filter:contact",
    "ignoreMultiRoomBin": true,
    "hideStuckSensor": false,
    "disableMultiRoom": false,
    "platform": "iRobotPlatform"
}
```

### Node Version:

v20.18.0

### NPM Version:

10.8.2

### Homebridge Version:

1.8.5

### Plugin Version:

3.2.3

### Operating System:

Raspberian

**Comment by taurgis** [comment]
I have confirmed the issue and released an ALPHA version that should solve this issue. Please let me know if it works for you, and I'll proceed to make it the official "non-alpha" release.

**Comment by WotAGitUR** [comment]
That worked, thank you. I now see this in the log :)

```
[24/11/2024, 10:51:41] [iRobot] Initializing iRobotPlatform platform...
[24/11/2024, 10:51:42] [iRobot] Using manual discovery as per config
[24/11/2024, 10:51:42] [iRobot] Configuring roomba: RobotS9
[24/11/2024, 10:51:43] [iRobot] Adding new accessory: RobotS9
[24/11/2024, 10:51:43] [iRobot] Configured Roomba Accessory: RobotS9
```

**Comment by taurgis** [comment]
Released 3.2.4, moving it out of alpha
